doc/developer/projects.rst

```diff
@@ -32,15 +32,50 @@ Pedagogical Interactive Notebooks for Algorithms Implemented in NetworkX
 
 - Complexity: Depending on the algorithms you are interested to work on.
 
-- Interested Mentors: `@dschult <https://github.com/dschult/>`__,
-  `@MridulS <https://github.com/MridulS/>`__,
+- Interested Mentors: `@MridulS <https://github.com/MridulS/>`__,
   `@rossbar <https://github.com/rossbar/>`__
   
 - Expected time commitment: This project can be either a medium project (~175 hours)
   or a large project (~350 hours). The contributor is expected to contribute 2-3
   pedagogical interactive notebooks for the medium duration project and 4-5 notebooks
   for the long duration project.
 
+Visualization API with Matplotlib
+---------------------------------
+
+- Abstract: NetworkX has some basic drawing tools that use Matplotlib to render the
+  images. The API hasn't changed while Matplotlib has changed. Also we have added or
+  are trying to add new features especially with regard to plotting edges. We'd like
+  someone to read a lot about what we offer and also what Matplotlib offers, and
+  come up with a nice way for users to draw graphs flexibly and yet with good defaults.
+  There is little chace just a broad topic could be completed in one summer, but a
+  roadmap and substantial headway on that road is possible.
+
+- Recommended Skills: Python, matplotlib experience.
+
+- Expected Outcome: A roadmap for a refined API for the matplotlib tools within NetworkX
+  as well as code in the form of PR(s) which implement (part of) that API with tests.
+
+- Interested Mentors: `@dschult <https://github.com/dschult/>`__,
+
+Incorporate a Python library for ISMAGs isomorphism calculations
+----------------------------------------------------------------
+
+- Abstract: A team from Sandia Labs has converted the original java implementation of
+  the ISMAGS isomorphism routines to Python. They have invited us to incorporate that
+  code into NetworkX if we are interested. We'd like someone to learn the ISMAGS code
+  we currently provide, and the code from this new library and figure out what the
+  best combination is to include in NetworkX moving forward. That could be two separate
+  subpackages of tools, or more likely a combination of the two sets of code, or a
+  third incantation that combines good features from each.
+
+- Recommended Skills: Python, graph algorithms.
+
+- Expected Outcome: A plan for how to best incorporate ISMAGS into NetworkX along
+  with code to do that incorporation.
+
+- Interested Mentors: `@dschult <https://github.com/dschult/>`__,
+
```

Comment:
Both sound perfect!
Could you also add a section on `Expected time commitment` as then we will be able to use these as GSoC ideas too. There are supposed to be either 350 or 175 hours projects.

Reply:
I think both of these projects could be even more than full time for 10 weeks, but that's what I put because both outreachy and GSOC are 10 weeks. It seems like it would be harder to make enough progress in the 175 hrs category.